src/renderer/components/Dex.vue

```diff
@@ -57,14 +57,6 @@ export default {
 
     this.$services.dex.completeSystemAssetWithdrawals();
 
-    this.$store.state.socket.opened = () => {
```

Comment:
This needed to be moved to SOCKET_RECONNECT mutation. This resulted in double subscriptions.